### PR TITLE
Update heuristic for BIDS file naming and add README

### LINKED ISSUE #32: Add PPMI-specific README
The `README.md` in the root directory is inherited from the template `mr_proc` repository and should not be updated with dataset-specific information. [Here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes) it says that a README file in `.github` would take precedence over one in the root directory, which in turn takes precedence over one in a `docs` subdirectory. So we can move the generic README in the `mr_proc` template directory to a `docs` subdirectory, then have the dataset-specific READMEs in the root directory.

The PPMI-specific README should have:

  - What the different BIDS tags mean in the filenames

### PR DESCRIPTION
**Pull request by michellewang**
Closes #31, closes #32.

- Update filters for PPMI description strings
  - Neuromelanin scans are T1s
  - Added new description strings from most recent LONI download
- Replace old `heuristic.py` with new rules for `anat` and `dwi` images
- Add PPMI `README` to be shown on main GitHub repo page
- Add `NOTES.md` for keeping track of PPMI-specific issues/quirks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# `mr_proc`: Parkinson's Progression Markers Initiative dataset
+
+This repository contains code to process tabular and imaging from the Parkinson's Progression Markers Initiative (PPMI) dataset. It is a fork of the main [`mr_proc`](https://github.com/neurodatascience/mr_proc) repository. `mr_proc` is a lightweight workflow management and harmonization tools for MRI and clinical data. This fork adds scripts and configuration files that are specific to PPMI.
```
Comment (nikhil153) [inline README.md:3]: - This repository contains code to process tabular and imaging `data` ? 
- This fork adds scripts, configuration files, and `downstream analyses` that are specific to PPMI.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- `anat`
+  - The available suffixes are: `T1w`, `T2w`, `T2starw`, and `FLAIR`
+  - Most images have an `acq` tag:
+    - Non-neuromelanin images: `acq-<plane><type>`, where
+        - `<plane>` is one of: `sag`, `ax`, or `cor` (for sagittal, axial, or coronal scans respectively)
+        - `<type>` is one of: `2D`, or `3D`
+    - Neuromelanin images: `acq-NM`
+  - For some images, the acquisition plane (`sag`/`ax`/`cor`) or type (`2D`/`3D`) cannot be easily obtained. In those cases, the filename will not contain an `acq` tag.
+- `dwi`
+  - All files have the `dwi` suffix.
```
Comment (nikhil153) [inline README.md:21]: All imaging files*

File: tabular/ppmi_imaging_ignored.csv [hunk 3]
```diff
@@ -50,6 +51,7 @@ IsoADC
 LOCALIZER
 LOWER 2D GRE MT
 MIDLINE SAG LOC
+MPR - SmartBrain
```
Comment (nikhil153) [inline tabular/ppmi_imaging_ignored.csv:54]: Smart brain!

File: workflow/bids_conv/heuristic.py [hunk 4]
```diff
@@ -1,182 +1,239 @@
...
+        'AX DTI 32 DIR FAT SHIFT L NO ANGLE',
+        'AX DTI _reverse', # one subject has this and 'AX DTI _RL'
+
+    ],
+    'RL': [
+        '2D DTI EPI FAT SHIFT RIGHT',
+        'AX DTI 32 DIR FAT SHIFT R',
+        'AX DTI 32 DIR FAT SHIFT R NO ANGLE'
+    ]
+}
```
Comment (nikhil153) [inline workflow/bids_conv/heuristic.py:50]: Why no DIR_DESCRIPTION_MAP for AP / PA with 64 directions and R-L with 30 directions scans? Is that taken care by regex?

**Review by nikhil153 — APPROVED** [review]
Made a few minor comments and one clarification comment on DIR_DESCRIPTION_MAP. Should be good to go is it's just my confusion!

File: workflow/bids_conv/heuristic.py (hunk 4 above)
Comment (nikhil153) [reply workflow/bids_conv/heuristic.py]: I see - what about the `"DTI_30dir R-L` one?

Comment (michellewang) [reply workflow/bids_conv/heuristic.py]: `DTI_30dir R-L` is also handled by the regex, anything with `R`...`L` or `L`...`R` should be handled, it's just the ones with a single side (e.g., `LEFT`, `L` without an `R`) that need special handling

**Pull request merged**